tests/integration/test_plot_gen.py

```diff
@@ -0,0 +1,37 @@
+from vega_sim.scenario.curve_market_maker.scenario import CurveMarketMaker
+from vega_sim.null_service import VegaServiceNull
+from vega_sim.tools.scenario_plots import plot_run_outputs
+
+
+def generate_trading_plot():
```

Comment:
Is this missing `@pytest.mark.integration` or maybe start with the worst `test`.  My `pytest` doesn't recognise this as a test.

Reply:
It is very specifically not doing either 😄. I don't actually want it picked up by pytest runs at all since it's really just generating a test plot. I'll add some documentation explaining what's up though